src/index.tsx

```diff
@@ -73,8 +73,8 @@ i18next
     ]
   })
 
-const themeClass = (configStore.get('theme') as string) || ''
-document.body.classList.add(themeClass)
+const themeClass = configStore.get('theme') as string
```

Comment:
let's keep a ` || 'default'` here, so the class matches the value in the select tag of ThemeSelector, so it's selected properly